src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+spring:
+  application:
+    name: acquisitions-tools
+  datasource:
+    username: ${DB_USERNAME:folio_admin}
+    password: ${DB_PASSWORD:folio_admin}
+    url: jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5432}/${DB_DATABASE:okapi_modules}
+    driver-class-name: org.postgresql.Driver
+
+server:
+  port: 8080
+
+folio:
+  login:
+    url: ${KONG_URL}
+  api:
+    url: ${FOLIO_URL}
+    tenant: consortium
```

Comment:
it can be different tenantId for central tenant, so should be as env variable

Reply:
changed to env variables